pkg/api/types.go

```diff
@@ -1205,6 +1208,21 @@ type KeyToPath struct {
 	Mode *int32
 }
 
+// LocalVolume represents a local volume accessible only from one node
+type LocalVolumeSource struct {
+	// NodeName represents the node that the volume is located at
+	NodeName string
+	// Fs represents a filesystem-based local volume
+	// +optional
+	Fs *LocalFsVolume
+}
+
+// LocalFsVolume represents a filesystem-based local volume
+type LocalFsVolume struct {
```

Comment:
Yeah, consistency with the rest of the volume plugins is important too, so when we do eventually add block support, then we can handle all the volume plugins the same way.  @jsafrane, wdyt?